backend/routes/graph.py

```diff
@@ -406,20 +386,7 @@ def load_relationship_graph(save_if_not_exists=True):
     #   import builtins
     #   builtins.DONT_LOAD_GRAPH = True
     import builtins
-
     if hasattr(builtins, 'DONT_LOAD_GRAPH') and builtins.DONT_LOAD_GRAPH:
         warnings.warn('not loading relationship graph')
     else:
-        REL_GRAPH = load_relationship_graph(save_if_not_exists=True)
-        # REVERSE_GRAPH = REL_GRAPH.reverse()
-
-        # G_ROOTS = set([n for n in REL_GRAPH.nodes if REL_GRAPH.in_degree(n) == 0])
-        # def distance_to_root(G, node):
-        #     n = node
-        #     d = 0
-        #     for p in G.predecessors(node):
-        #         if n in G_ROOTS:
-        #             return d
-        #         d += 1
-        #         n = p
-        #     raise Exception(f"can't find root for {node}")
+        REL_GRAPH = load_relationship_graph()
```

Comment:
## Activates: On server start
Since this file gets imported during server start, this line runs.